### PR TITLE
fix: Add Sentry DSN

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Add Sentry DSN for GitHub

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Test locally on your own infrastructure
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation)) 
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

**Review by hermanschaaf — APPROVED** [review]

**Comment by erezrokah** [comment]
@candiduslynx I'll let you merge this when it's convenient to you

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**
